client/Application.jsx

```diff
@@ -81,16 +81,6 @@ class App extends React.Component {
             this.props.receiveJoinGame(game);
         });
 
-        socket.on('leavegame', (game, player) => {
-            var isMe = false;
-
-            if(player === this.props.username) {
-                isMe = true;
-            }
-
-            this.props.receiveLeaveGame(game, isMe);
```

Comment:
`receiveLeaveGame` isn't being used anywhere now. Remove the React property for the handler.